test/alias.js

```diff
@@ -39,6 +39,9 @@ describe("alias", function () {
 				recursive: "recursive/dir",
 				"/d/dir": "/c/dir",
 				"/d/index.js": "/c/index",
+				// alias configuration should work
+				"#": "/c/dir",
+				"@": "/c/dir",
```

Comment:
maybe also add a test case below for that

Reply:
technically it fails without applying.. (just setting crash resolving) lets add usage anyway.